python-frontend/src/main/java/org/sonar/plugins/python/api/PythonInputFileContext.java

```diff
@@ -59,7 +64,7 @@ public CacheContext cacheContext() {
 
   @Beta
   public Collection<Symbol> stubFilesSymbols() {
-    return TypeShed.stubFilesSymbols();
+    return projectLevelSymbolTable.typeShedDescriptorsProvider().stubFilesSymbols(projectLevelSymbolTable);
```

Comment:
Note: I notice it's a bit weird to pass the `projectLevelSymbolTable` to the `typeShedDescriptorsProvider` when it is itself a part of the `projectLevelSymbolTable`. However, given it is code meant to disappear after we finish the removal of the V1 engine, I think it's okay to keep as it is.

Reply:
why not delegate this method inside the `ProjectLevelSymbolTable` to hide it behind the scenes?